br/pkg/lightning/backend/local/localhelper.go

```diff
@@ -399,6 +399,18 @@ func (local *Backend) SplitAndScatterRegionByRanges(
 	return nil
 }
 
+// ScatterRegion scatter the regions and retry if it fails. It returns error if can not scatter after max_retry.
+func (local *Backend) ScatterRegion(ctx context.Context, regionInfo *split.RegionInfo) error {
+	backoffer := split.NewWaitRegionOnlineBackoffer().(*split.WaitRegionOnlineBackoffer)
+	_ = utils.WithRetry(ctx, func() error {
+		return (local.splitCli.ScatterRegion(ctx, regionInfo))
+	}, backoffer)
+	if ctx.Err() != nil {
```

Comment:
i think we need check err of `utils.WithRetry` here

Reply:
I don't understand. Won't ctx.err() be set if utils.WithRetry fails. Or are you asking to print error inside utils.withretry.

Reply:
got it